eiffel-vocabulary/EiffelArtifactPublishedEvent.md

```diff
@@ -24,6 +24,11 @@ __Type:__ Object[]
 __Required:__ Yes  
 __Description:__ A list of locations at which the artifact may be retrieved.
 
+#### data.locations.name
+__Type:__ String  
+__Required:__ No  
+__Description:__ Identifies the name of the file within the artifact for which this location provides the URI. Must correspond to a __data.fileInformation.name__ value in the EiffelArtifactCreatedEvent event connected via the __ARTIFACT__ link.
```

Comment:
Add link to EiffelArtifactCreatedEvent and remove the "event" word after. Right now it reads as "..Event event.."

Reply:
Done.